api/swagger.yaml

```diff
@@ -547,6 +586,15 @@ paths:
               $ref: '#/definitions/SpecDiffTime'
         default:
           $ref: '#/responses/UnknownError'
+          
+  /features:
+    get:
+      summary: 'Get the list of APIClarity features'
```

Comment:
I saw that there is an unused feature operation enum (APIClarityFeatureOperationEnum). is this list of enabled (added) features, or list of all features known to the system?
Should this be "Get the list of APIClarity active features"?

Reply:
This is the list of features known by the system , so for a given running instance of APIClarity the list is always the same.

OperationEnum is a left over of the PUT operation and I am removing it